pkg/daemon/journal.go

```diff
@@ -0,0 +1,61 @@
+package daemon
+
+import (
+	"fmt"
+	"net/http"
+
+	"github.com/testground/testground/pkg/api"
+	"github.com/testground/testground/pkg/logging"
+)
+
+func (d *Daemon) getJournalHandler(engine api.Engine) func(w http.ResponseWriter, r *http.Request) {
```

Comment:
I also don't like the `journal` name. I am a bit hesitant to call it `RunnerOutput`, because it sounds like `run output` and this is something else... could be the `outputs` we collect or the `api.RunOutput`...